models/pull.go

```diff
@@ -185,7 +185,7 @@ func (pr *PullRequest) GetDefaultMergeMessage() string {
 		log.Error("Cannot load issue %d for PR id %d: Error: %v", pr.IssueID, pr.ID, err)
```

Comment:
I would like to confirm is the head repository is the same as the base repository. Then return different message. When that is the same repository, we could ignore `%s/%s:`

Reply:
@lunny take a look I've just pushed some changes up.